.travis.yml

```diff
@@ -9,39 +9,47 @@ matrix:
       env:
         - USE_SYSTEM=1
         - USE_PROJECT=1
+        - USE_CGAL=1
+        - USE_VTK=1
     - os: osx
       compiler: clang
       env:
         - USE_SYSTEM=1
         - USE_PROJECT=1
         - BLASLAPACK_IMPLEMENTATION=vecLib
+        - BUILD_DOCUMENTATION=1
     - os: osx
       compiler: clang
       env:
         - USE_PROJECT=0
-        - USE_SYSTEM=0
         - BLASLAPACK_IMPLEMENTATION=Auto
-    - os: osx
+    - os: linux
```

Comment:
I read:

```
    - os: osx
      compiler: clang
      env:
        - USE_SYSTEM=1
        - USE_PROJECT=1
        - USE_CGAL=1
        - USE_VTK=1
    - os: osx
      compiler: clang
      env:
        - USE_SYSTEM=1
        - USE_PROJECT=1
        - BLASLAPACK_IMPLEMENTATION=vecLib
        - BUILD_DOCUMENTATION=1
    - os: osx
      compiler: clang
      env:
        - USE_PROJECT=0
        - BLASLAPACK_IMPLEMENTATION=Auto
```

personally I find it hard to know what BLASLAPACK implem is used for the three. Is Auto the same as not saying anything? if so does it mean that the same BLASLAPACK is tested twice?